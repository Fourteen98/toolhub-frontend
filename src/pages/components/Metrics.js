import React from 'react';
import { Pie } from 'react-chartjs-2';
import Chart from 'chart.js/auto'; // eslint-disable-line

const Metrics = ({ chartsData }) => (
  <>
    <Pie data={chartsData} />
  </>
);

export default Metrics;
