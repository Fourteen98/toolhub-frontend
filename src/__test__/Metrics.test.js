import React from "react";
import renderer from "react-test-renderer";
import Metrics from "../pages/components/Metrics";

describe("Metrics", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Metrics />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
