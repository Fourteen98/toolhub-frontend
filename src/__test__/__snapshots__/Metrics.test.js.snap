// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Metrics renders correctly 1`] = `
<canvas
  height={150}
  role="img"
  width={300}
/>
`;
